pkg/bkop/restore.go

```diff
@@ -93,6 +93,8 @@ func (o operator) LoadBinlog(ctx context.Context, dir string, restorePoint time.
 	binlogCmd.Stderr = os.Stderr
 	env := os.Environ()
 	env = append(env, "TZ=Etc/UTC")
+	// mysqlbinlog requires enough space to be specified as TMPDIR.
+	env = append(env, fmt.Sprintf("TMPDIR=%s", tmpDir))
```

Comment:
In this case, `+` is enough.
```suggestion
	env = append(env, "TMPDIR=" + tmpDir)
```